### PR TITLE
WAL-152, WAL-148:  ITN prep

### PR DESCRIPTION
**Pull request by remon-nashid**

### CODE REVIEW

File: packages/core/src/constants.ts [hunk 1]
```diff
@@ -89,7 +86,9 @@ export const defaultNetworkState: NetworkState = {
 export const uidProvidersWhitelist = [
   'https://polymathnetwork.github.io',
   'https://itn-polymesh.fractal.id',
-  'https://staging.itn-polymesh.fractal.id'
+  'https://staging.itn-polymesh.fractal.id',
+  'https://tokenstudio.polymesh.live',
```
Comment (FahdW) [inline packages/core/src/constants.ts:90]: change this to https://alcyone-tokenstudio.polymesh.live

File: packages/core/src/constants.ts [hunk 2]
```diff
@@ -89,7 +86,9 @@ export const defaultNetworkState: NetworkState = {
...
+  'https://dashboard.polymesh.live'
```
Comment (FahdW) [inline packages/core/src/constants.ts:91]: change this to https://alcyone-dashboard.polymesh.live

**Review by FahdW — CHANGES_REQUESTED** [review]

File: packages/ui/src/Popup/ProvideUidRequests/Request.tsx [hunk 3]
```diff
@@ -150,8 +150,8 @@ function Request ({ isFirst, reqId, request, url }: Props): React.ReactElement<P
               </Flex>
               <Text color='gray.1'
                 variant='b2m'>
-                You already have an existing uID assigned to the current Polymesh identity, accepting this uID will
-                overwrite the exisiting one. Do you like to accept?
+                You already have an existing uID assigned to the current Polymesh Account, accepting this uID will
+                overwrite the existing one. Do you like to accept?
```
Comment (F-OBrien) [inline packages/ui/src/Popup/ProvideUidRequests/Request.tsx:154]: This doesn't sound right to me. I think it should be "Do you accept?" or "Would you like to accept?" Or even "Do you want to proceed?"

Comment (remon-nashid) [reply packages/ui/src/Popup/ProvideUidRequests/Request.tsx:154]: Good catch! Definitely this needs to be changed to "Would you like to accept?". Thanks!

**Review by wahidrahim — APPROVED** [review]

**Comment by remon-nashid** [comment]
Thanks @wahidrahim

**Pull request merged**